app/src/main/java/org/mozilla/focus/notification/RocketMessagingService.java

```diff
@@ -7,29 +7,23 @@
 
 import android.app.PendingIntent;
 import android.content.Intent;
-import android.net.Uri;
 import android.support.v4.app.NotificationCompat;
 
-import org.mozilla.focus.activity.MainActivity;
 import org.mozilla.focus.telemetry.TelemetryWrapper;
+import org.mozilla.rocket.component.RocketLauncherActivity;
 
 // Prov
 public class RocketMessagingService extends FirebaseMessagingServiceWrapper {
 
     //
     @Override
-    public void onRemoteMessage(String url, String title, String body) {
+    public void onRemoteMessage(Intent intent, String title, String body) {
         if (!TelemetryWrapper.isTelemetryEnabled(this)) {
             return;
         }
+        // RocketLauncherActivity will handle this intent
+        intent.setClass(getApplicationContext(), RocketLauncherActivity.class);
```

Comment:
How about `intent.setClassName(this, AppConstants.LAUNCHER_ACTIVITY_ALIAS);`?

Reply:
thx~!